docs/content/preview/admin/yb-admin.md

```diff
@@ -1114,32 +1114,38 @@ $ curl -s http://<any-master-ip>:7000/cluster-config
 
 #### set_preferred_zones
 
-Sets the preferred availability zones (AZs) and regions.
+Sets the preferred availability zones (AZs) and regions. Tablet leaders are placed in alive and healthy nodes of the highest preference zones (preference 1). When no such node is available, then alive and healthy nodes from the next highest preferred zone are picked. Zones with no preference are equally eligible to host tablet leaders.
 
 {{< note title="Note" >}}
 
 * Make sure you've already run [`modify_placement_info`](#modify-placement-info) command beforehand.
 
-* When nodes in the "preferred" availability zones and regions are alive and healthy, the tablet leaders are placed on nodes in those zones and regions.
-By default, all nodes are eligible to have tablet leaders. Having all tablet leaders reside in a single region will reduce the number of network hops for the database to write transactions and thus increase performance and lowering latency.
+* Having all tablet leaders reside in a single region will reduce the number of network hops for the database to write transactions and thus increase performance and lowering latency.
 
 * By default, the transaction tablet leaders will not respect these preferred zones and will be balanced across all nodes. In the transaction path, there is a roundtrip from the user to the transaction status tablet serving the transaction - if the leader closest to the user is used rather than forcing a roundtrip to the preferred zone, then there will be efficiency improvements.
 
+* Leader blacklisted nodes will not host any leaders irrespective of their preference.
+
+* affinitized_leaders in cluster configuration may be replaced with multi_affinitized_leaders.
```

Comment:
what are affinitized leaders? not previously described anywhere

Reply:
Will switch to 
"Cluster configuration stores preferred zones in either affinitized_leaders or multi_affinitized_leaders object"